Mastodon/Scene/Account/AccountListViewModel.swift

```diff
@@ -103,25 +103,25 @@ extension AccountListViewModel {
         authentication: MastodonAuthentication,
         activeAuthentication: MastodonAuthentication
     ) {
-        guard let user = authentication.user(in: context) else { return }
-        
+        guard let account = authentication.account() else { return }
```

Comment:
For me this method returns `nil` thus bailing the user out here and leaving the app in a dangling (everything is blank) state.

Reply:
I will look into that, thank you!

Reply:
Should work again :)

Reply:
Yes works now, thank you 👍